Client/game_sa/CPoolSAInterface.h

```diff
@@ -114,8 +114,9 @@ class CPoolSAInterface
 
     void Delete(uint index) { Release(index); }
 
-    bool IsEmpty(std::int32_t objectIndex) { return m_byteMap[objectIndex].bEmpty; }
-    bool IsContains(uint index)
+    std::int32_t Size() const noexcept { return m_nSize; };
+    bool IsEmpty(std::int32_t objectIndex) const { return m_byteMap[objectIndex].bEmpty; }
+    bool IsContains(uint index) const
```

Comment:
if above then `noexcept`

Reply:
Can cause exception. It's better to avoid it here for additional debug information